cinema/views.py

```diff
@@ -34,6 +38,33 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(str_id) for str_id in qs.split(",")]
+
```

Comment:
let's move it to function, as you want to use it in MovieSession ViewSet class too